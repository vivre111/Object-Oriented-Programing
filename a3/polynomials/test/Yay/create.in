create b 3/5 3 -2/1 0
print b 
create a 2/3 1 -2/4 6 9/1 0
print a
create a 2/3 6 -2/5 4 3/2 2 5/7 0
print a
print b
print a
create d 2/5 9 2/5 4 -0/4 1
print d
print a
