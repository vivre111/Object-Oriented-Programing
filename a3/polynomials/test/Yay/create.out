(3/5)x^3 + (-2)
(2/3)x + (9)
(2/3)x^6 + (-2/5)x^4 + (3/2)x^2 + (5/7)
(3/5)x^3 + (-2)
(2/3)x^6 + (-2/5)x^4 + (3/2)x^2 + (5/7)
(2/5)x^9 + (2/5)x^4
(2/3)x^6 + (-2/5)x^4 + (3/2)x^2 + (5/7)
